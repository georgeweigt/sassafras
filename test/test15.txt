* Two-way ANOVA with missing cells from STAT 650

data
input a b y;
datalines
1 1 15
1 1 16
1 1 14
1 2 18
1 2 19
1 3 .
2 1 16
2 1 15
2 2 .
2 3 21
2 3 22
2 3 22
3 1 .
3 2 19
3 2 20
3 3 22
3 3 24
3 3 23
;

proc reg
model y = a b
