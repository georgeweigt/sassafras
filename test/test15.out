                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      2          141.84242        70.92121      120.02     0.0000
     Error     12            7.09091         0.59091                       
     Total     14          148.93333                                       

              Root MSE            0.76871     R-Square     0.9524
              Dependent Mean     19.06667     Adj R-Sq     0.9445
              Coeff Var           4.03168                        

                              Parameter Estimates

                        Estimate     Std Err     t Value     Pr > |t|
          Intercept     11.10909     0.57990       19.16       0.0000
          A              0.69091     0.29682        2.33       0.0382
          B              3.18182     0.28386       11.21       0.0000

